dkist/dataset/utils.py

```diff
@@ -53,6 +54,8 @@ def dataset_info_str(ds_in):
         s += f"consists of {nframes} frames.\n"
         s += f"Files are stored in {ds.files.basepath}\n"
 
+    s += f"\nThis calibration has Dataset ID {dsID} and the original frames have Product ID {pID}.\n"
```

Comment:
We might want to wordsmith on "original frames".

@fraserwatson @eigenbrot ?

Reply:
Yeah I wasn't happy with that wording but wasn't really sure what else to put.

Reply:
We tend to refer to those frames internally as either "raw" or "Level 0".

Reply:
I like "input observe" frames. It is important that we say the product ID only referes to the observe frames.